_spinnaker/terraform_integration.md

```diff
@@ -9,15 +9,16 @@ Note that the Terraform integration is in beta while we work on improving the us
 {:.no_toc}
 ## Overview
 
```

Comment:
Somewhere, we probably want to add that this is behind a feature flag in 2.17.4.  In `~/.hal/default/profiles/settings-local.js`, add the line `window.spinnakerSettings.feature.terraform = true;`

Reply:
@ethanfrogers mentioned something about removing the feature flag this sprint. Easy addition depending on when we finish reviewing this. I can also remove the `Profile` stuff too depending on when that's going out.